ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -354,10 +354,10 @@ pattern CardanoNodeToClientVersion7 =
     HardForkNodeToClientEnabled
       HardForkSpecificNodeToClientVersion2
       (  EraNodeToClientEnabled ByronNodeToClientVersion1
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion3
```

Comment:
This seems fine. Reasoning: we know 7 isn't immutable yet, since it's the one that introduced Alonzo and so obviously hasn't been publicly released.